NEWS.md

```diff
@@ -1,6 +1,26 @@
 # ghactions4r development
 * Add option to use Air instead of styler in the doc-and-styler-r workflow.
 
+# ghactions4r 1.0.0
+
+This is technically the first stable release of ghactions4r! The API will not change unexpectedly.
+
+## Major Changes
+
+* Allow style and doc to work on multiple branches and make PRs for each branch in https://github.com/nmfs-ost/ghactions4r/pull/193 

```

Comment:
nitpick: I noticed “style and doc,” “doc and style,” and “doc-and-style” are all used in the release notes. Maybe choose one consistent form.

Reply:
Great point! I corrected this.